pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -141,6 +141,7 @@ public class ServiceUnitStateChannelImpl implements ServiceUnitStateChannel {
     private volatile long lastOwnEventHandledAt = 0;
     private long lastOwnedServiceUnitCountAt = 0;
     private int totalOwnedServiceUnitCnt = 0;
+    private volatile boolean disablePubOwnedEvent = false;
```

Comment:
Can we add this case in the ChannelState enum below? I think a better name could be "CleaningOwnerships"

Reply:
Actually I found there is a `Disabled` state introduced in https://github.com/apache/pulsar/pull/23301. But it's never set other than tests.

Reply:
I used this state in the latest code now